plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryType.java

```diff
@@ -335,6 +338,7 @@ public Type getNativeType(BigQueryType.Adaptor typeAdaptor)
     {
         switch (this) {
             case NUMERIC:
+            case BIGNUMERIC:
```

Comment:
Should there be a comment that BIGNUMERIC without precision and scale are filtered in `getColumns`?